game-core/src/main/java/tools/map/making/MapProperties.java

```diff
@@ -3,51 +3,20 @@
 import java.awt.Color;
 import java.util.List;
 import java.util.Map;
-import java.util.Map.Entry;
 import java.util.TreeMap;
 
 import games.strategy.engine.data.properties.PropertiesUi;
 import games.strategy.triplea.Constants;
-import games.strategy.triplea.image.UnitImageFactory;
-import games.strategy.triplea.ui.mapdata.MapData;
 import games.strategy.util.Tuple;
 
 /**
  * An object to hold all the map.properties values.
  */
-public class MapProperties {
```

Comment:
Pretty sure the bulk of this class is accessed reflectively by the Map Properties Maker tool.

Reply:
I thought so as well, on a double check you are right @ssoloff , here it is:

```
    private String getOutPutString() {
      final StringBuilder outString = new StringBuilder();
      for (final Method outMethod : mapProperties.getClass().getMethods()) {
        final boolean startsWithSet = outMethod.getName().startsWith("out");
        if (!startsWithSet) {
          continue;
        }
        try {
          outString.append(outMethod.invoke(mapProperties));
        } catch (final IllegalArgumentException | InvocationTargetException | IllegalAccessException e) {
          log.log(Level.SEVERE, "Failed to invoke method reflectively: " + outMethod.getName(), e);
        }
      }
      return outString.toString();
    }
  }

```

That is pretty interesting code as only methods with an `out` prefix are used by reflection. Taking this into account, I rolled back the updates to `MapProperties.java` and `*Maker.java`:
- e05930636
- 6acfc5904518e7b760d3481b1f102b51b4c59de7

Then I rolled forward properly removing unused and suppressing unused warnings on 'out*' methods:
- c20b9d5373bae8c1411cf8473250898850ddefa0

Reply:
I really appreciate the diligence @ssoloff , it is quite exemplary.

Map properties unused methods clearly needs to be done carefully, IMO probably best to leave that for a dedicated PR going forward. I reverted map property updates from this branch: c20b9d5

Reply:
Post-revert:
![screenshot from 2019-01-25 07-59-16](https://user-images.githubusercontent.com/12397753/51757001-2ece9b80-2077-11e9-9edb-e3a5e421d91c.png)